tests/FSharp.Test.Utilities/ILVerifierModule.fs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+namespace FSharp.Test
+
+open FSharp.Test.Compiler
+open System
+open System.IO
+open TestFramework
+
+[<AutoOpen>]
+module ILVerifierModule =
+    let config = initializeSuite ()
+
+    let private exec (dotnetExe: string) args workingDirectory =
+        let arguments = args |> String.concat " "
+        let exitCode, _output, errors = Commands.executeProcess dotnetExe arguments workingDirectory
+        let errors = errors |> String.concat Environment.NewLine
+        errors, exitCode
+
+    let private verifyPEFileCore peverifierArgs (dllFilePath: string) =
+        let nuget_packages =
+            match Environment.GetEnvironmentVariable("NUGET_PACKAGES") with
+            | null ->
+                let profile = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile)
+                $"""{profile}/.nuget/packages"""
+            | path -> path
+        let peverifyFullArgs = [ yield "exec"; yield $"""{nuget_packages}/dotnet-ilverify/9.0.0/tools/net9.0/any/ILVerify.dll"""; yield "--verbose"; yield dllFilePath; yield! peverifierArgs ]
+        let workingDirectory = Path.GetDirectoryName dllFilePath
+        let _, exitCode =
+            let peverifierCommandPath = Path.ChangeExtension(dllFilePath, ".peverifierCommandPath.cmd")
+            let args = peverifyFullArgs |> Seq.fold(fun a acc -> $"{a} " + acc) ""
+            File.WriteAllLines(peverifierCommandPath, [| $"{args}" |] )
+            File.Copy(typeof<RequireQualifiedAccessAttribute>.Assembly.Location, Path.GetDirectoryName(dllFilePath) ++ "FSharp.Core.dll", true)
```

Comment:
We are passing a full path to FSharp.Core as a reference, but on top of that we still have to copy it locally?
(despite the path from the --reference args points to a different file?)

Reply:
Actually, the paths should be the same. typeof<RequireQualifiedAccessAttribute>.Assembly.Location and typeof<unit>.Assembly.Location should resolve to the same value.  But I am wondering if the copy is strictly necessary.  It probably is for execution of the built testcase.  But that is not a concern of verification.

Reply:
Yeah, it is not necessary when running the test under the framework, it is necessary when executing the test directly from the command line., I will remove it.

Reply:
@t-gro, that is a good question.  The path to the file is the same in both cases. However, the copy is only necessary if you try to execute the app directly from the command time. Which we don't when executing from the test framework. Anyway I removed the copy.